test/fixtures-import.js

```diff
@@ -39,7 +39,7 @@ const verify = (bibfile, processComments) => {
 
     let expected = path.join(path.dirname(bibfile), name + ".json")
     // Uncomment the following line to save the results as expected test results.
-    //fs.writeFileSync(expected, JSON.stringify(found, null, 4) + '\n')
+    // fs.writeFileSync(expected, JSON.stringify(found, null, 2))
```

Comment:
@DerDrodt given that your output is 4-spaced and there is the final line ending, I guess this line wasn't activated to create the test output?

I think it would make sense to just keep the line as it currently is in the master branch.

Reply:
Yes. Fixed now.